src/Tribe/Aggregator/Page.php

```diff
@@ -420,6 +420,13 @@ public function register_menu_item() {
 			[ $this, 'render' ]
 		);
 
+		/**
+		 * Fires an Action after the submenu item is created.
+		 *
+		 * @since TBD
+		 */
+		do_action( 'tec_events_after_import_submenu_created' );
```

Comment:
Instead of doing that cant we create the page using our admin pages and adjust its placement with the priority param ?

Reply:
I don't know why I didn't try using the action priority. I was experimenting with the `position` property, which didn't seem to do anything. Using priority 11 worked!